presto-docker-image/build-local.sh

```diff
@@ -4,14 +4,28 @@ set -euxo pipefail
 
 # Retrieve the script directory.
 SCRIPT_DIR="${BASH_SOURCE%/*}"
+cd ${SCRIPT_DIR}
 
-# Move to the root directory, as that
-# is where the Dockerfile resides.
-cd ${SCRIPT_DIR}/..
-
+# Move to the root directory to run maven for current version.
+pushd ${SCRIPT_DIR}/..
 PRESTO_VERSION=$(./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}')
+popd
+
+if [[ -d tmp ]]; then
```

Comment:
It might be better to use a new random temporary directory
```java
WORK="$(mktemp -d)"

Reply:
👍 This requires us to pass the temp directory to the docker build, but I think that's ok.